source/fundamentals.txt

```diff
@@ -30,6 +30,7 @@ Fundamentals
    Time Series Collections </fundamentals/time-series>
    In-Use Encryption </fundamentals/encrypt-fields>
    Search Geospatially </fundamentals/geo>
+   Replica Set Operations </fundamentals/read-write-configuration>
```

Comment:
Should this display on this page in the [deploy preview](https://deploy-preview-322--mongodb-docs-csharp.netlify.app/fundamentals/) at the end of the list? I don't see it for some reason 🤔

Reply:
Forgot the link! Will add in revision.